ipatests/test_integration/util.py

```diff
@@ -17,51 +17,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import time
 import re
 
 from ipaplatform.paths import paths
 from ipalib.constants import DEFAULT_CONFIG
 
```

Comment:
Since we are moving the modules into a package, why do we keep this module around? I do not know the rationale for the util module in here but the functions could move to `ipatests/pytest_plugins/integration/tasks.py`, with appropriate changes to `test_legacy_clients`, `test_sudo` and `test_trust` modules.